xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/TransferFeeTest.java

```diff
@@ -35,4 +37,13 @@ public void percentValueIncorrectFormat() {
       "Percent value should have a maximum of 2 decimal places."
     );
   }
+
+  @Test
+  public void validateBounds() {
```

Comment:
Can we move the tests in `org.xrpl.xrpl4j.model.client.common.TransferFeeTests` into here?

Reply:
cool cool